queue.go

```diff
@@ -0,0 +1,72 @@
+/* Copyright (c) 2021 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package nebula_go
+
+import (
+	"container/list"
+	"sync"
+)
+
+// Queue goroutine safe for list
+type Queue struct {
```

Comment:
In computer science, a queue is a collection of entities that are maintained in a sequence and can be modified by the addition of entities at one end of the sequence and the removal of entities from the other end of the sequence.

Would it be better to use `SafeList`,`ConcurrentList` or other name?

refs: https://en.wikipedia.org/wiki/Queue_(abstract_data_type)

Reply:
ok for me

Reply:
Other list iterations may also have the same problems.